django/db/models/fields/related_descriptors.py

```diff
@@ -110,7 +110,8 @@ def get_queryset(self, **hints):
             if not getattr(related_model._default_manager, 'silence_use_for_related_fields_deprecation', False):
                 warnings.warn(
                     "use_for_related_fields is deprecated, instead "
-                    "set Meta.base_manager_name on '{}'.".format(related_model._meta.label),
+                    "set Meta.base_manager_name on '{}' then remove "
```

Comment:
I don't think "then remove use_for_related_fields" is needed. Removing usage of the deprecated feature is always implied by deprecation warnings. Can the two options not be used together though? That might be problematic for projects that want to support multiple versions of Django.